src/containers/CampaignList/components/CampaignHeader.tsx

```diff
@@ -0,0 +1,99 @@
+import { Divider, Typography } from "@material-ui/core";
+import Chip from "@material-ui/core/Chip";
+import { green, orange } from "@material-ui/core/colors";
+import CheckCircleOutlineRoundedIcon from "@material-ui/icons/CheckCircleOutlineRounded";
+import WarningRoundedIcon from "@material-ui/icons/WarningRounded";
+import React from "react";
+
+type IconMap = {
+  [key: string]: JSX.Element;
+};
+
+const CHIP_ICONS: IconMap = {
+  "Not Started": <WarningRoundedIcon style={{ color: orange[300] }} />,
```

Comment:
Suggestion: Add a matching "Started" icon or use the shaded rectangle for "Not Started" as well for consistency - see notion doc https://rewiredcoop.slack.com/archives/C02KUGE68UX/p1690904317810059?thread_ts=1690808489.583299&cid=C02KUGE68UX